{
	"id": "CVE-2023-37920",
	"vulnerabilities": [
		{
			"content": {
				"id": "CVE-2023-37920",
				"references": [
					{
						"source": "security.alpinelinux.org",
						"url": "https://security.alpinelinux.org/vuln/CVE-2023-37920"
					}
				]
			},
			"segments": [
				{
					"ecosystem": "alpine:3.18"
				}
			]
		}
	],
	"detections": [
		{
			"ecosystem": "alpine:3.18",
			"conditions": [
				{
					"criteria": {
						"operator": "OR",
						"criterions": [
							{
								"type": "version",
								"version": {
									"vulnerable": true,
									"fix_status": {
										"class": "fixed"
									},
									"package": {
										"name": "py3-certifi",
										"architectures": [
											"aarch64",
											"armhf",
											"armv7",
											"ppc64le",
											"s390x",
											"x86",
											"x86_64"
										],
										"repositories": [
											"main"
										]
									},
									"affected": {
										"type": "apk",
										"range": [
											{
												"lt": "2023.7.22-r0"
											}
										],
										"fixed": [
											"2023.7.22-r0"
										]
									}
								}
							}
						]
					}
				}
			]
		}
	],
	"data_source": {
		"id": "alpine-secdb",
		"raws": [
			"vuls-data-raw-alpine-secdb/3.18/main.json"
		]
	}
}
